src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+    private String color = getColor();

```

Comment:
not sure what is supposed to happen here - you're trying to initialize a default value for the color by calling the getter, which points to...?
Remove this assignment.

Reply:
Please fix this for other figures as well.

Reply:
ok, I corrected the lines. but I left the field 'color'...